build/ci.yml

```diff
@@ -10,7 +10,7 @@ pr:
 
 variables:
   Build.Major: 0
-  Build.Minor: 13
+  Build.Minor: 12
```

Comment:
Was this intentional?

Reply:
No, it was not, thanks! Should I change it to 14 while I am at it, since we will have breaking changes with the next release?

Reply:
I think keeping it in sync with the official builds from the `qdk` repo is most desirable, so we could probably wait on that (unless that is also imminent).